homeassistant/components/recorder/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "recorder",
   "name": "Recorder",
   "documentation": "https://www.home-assistant.io/integrations/recorder",
-  "requirements": ["sqlalchemy==1.3.13"],
+  "requirements": ["sqlalchemy==1.3.13", "orjson==2.5.0"],
```

Comment:
Let's add this to the main requirements of Home Assistant so that we can also add it to https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/aiohttp_client.py#L66-L71

```python
aiohttp.ClientSession(json_serialize=orjson.dumps)
```

Reply:
(as default serializer it's fine)

Reply:
Also add it to `util/json.py` https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/util/json.py#L31